src/style.css

```diff
@@ -1,3 +1,137 @@
-body {
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+font,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td {
   margin: 0;
 }
+
+:root {
+  font-family: Roboto, Helvetica, sans-serif;
+  box-sizing: border-box;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  background-color: bisque;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content:space-around;
+}
+
+.logo {
+  margin: auto 0 auto 50px;
+  height: 40px;
+}
+
+.nav {
+  align-items: baseline;
+}
+
+ul {
```

Comment:
Don't use tag names for styling. Instead, use classes to apply styles to elements. For example, you should assign a class to the `ul` element and style that class, rather than styling the `ul` tag directly.